jenkins/compy_pace.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash -xle
+# Purpose: Automate performance data upload to PACE
+# Author: Sarat Sreepathi (sarat@ornl.gov)
+
+# boiler: every script must have these three lines
+export SCRIPTROOT=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && cd .. && pwd )
+export CIME_MACHINE=compy
+#source $SCRIPTROOT/util/setup_common.sh
+
+# Performance archive directory on this platform
+PERF_ARCHIVE_DIR=/compyfs/performance_archive
+# Archives of old performance data on this platform
+OLD_PERF_ARCHIVE_DIR=/compyfs/OLD_PERF
+
+# The performance archive directory on the platform is the working dir
+cd ${PERF_ARCHIVE_DIR}
+
+if [ $? -ne 0 ]
+then
+  echo "Error:Unable to change working directory to ${PERF_ARCHIVE_DIR}"
+  exit 1
+fi 
+
+# Download PACE upload script
+# Python2 version (supported by system Python without requiring module load)
+wget -O pace-upload https://pace.ornl.gov/static/tools/pace-upload
+# Python3 version (requires Anaconda/Python module with requests package)
+#wget -O pace-upload https://pace.ornl.gov/static/tools/pace-upload3
+
+if [ $? -ne 0 ]
```

Comment:
Since you passed the `-e` flag to bash in the #! line, you don't need to check error codes manually. In fact, this code will never be executed because `-e` caused the script to immediately stop when any command returns a non-zero error code.